initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -101,26 +101,17 @@ fi
 
 if [ -e /etc/debian_version ]; then
     echo "Removing DEB"
-    if [ $force -eq 0 ]; then
-        dpkg --purge hpccsystems-platform
-        if [ $? -ne 0 ]; then
-           message dpkg
-           exit 1
-        fi
-    else
-        dpkg --purge --force-all hpccsystems-platform
+    apt-get remove -y hpccsystems-platform
+    if [ $? -ne 0 ]; then
+       message dpkg
```

Comment:
Yes.  Interestingly while you can't install a package that isn't on an external repository, apt uses it's database of installed packages (which dpkg updates) to figure out dependency issues when you go to do a remove.  So instead of it being like dpkg and throwing errors when it meets dependencies, it will just handle them for us.